src/plugins/hapi.js

```diff
@@ -57,6 +57,17 @@ function createWrapDispatch (tracer, config) {
 }
 
 module.exports = [
+  {
+    name: '@hapi/hapi',
+    versions: ['>=18.2'],
```

Comment:
I think this is also needed for 17.9+ https://www.npmjs.com/package/@hapi/hapi/v/17.9.0